widget/embedded/src/pages/WalletsPage.tsx

```diff
@@ -74,17 +87,44 @@ export function WalletsPage() {
   const selectedWalletState =
     selectedWallet?.state || WalletState.NOT_INSTALLED;
 
+  const filteredBlockchains = filterBlockchainsByWalletTypes(list, blockchains);
+  const countActiveCategories = getCountCategories(filteredBlockchains);
+  const showCategory = countActiveCategories !== 1;
+
+  const filteredWallets = list.filter((wallet) => {
+    if (blockchainCategory === BlockchainCategories.ALL) {
```

Comment:
this comparison is done here and also again in the `filterByType` function.

Reply:
this line of code was added to solve an issue where wallet skeleton isn't displayed and meta wasn't loaded yet.We also use filterByType for the blockchain page